ansible/configs/three-tier-app/default_vars_ec2.yml

```diff
@@ -12,9 +12,9 @@ aws_region: us-east-1
 ## Networking (AWS)
 subdomain_base_short: "{{ guid }}"
 subdomain_base_suffix: ".example.opentlc.com"
-subdomain_base: "{{subdomain_base_short}}{{subdomain_base_suffix}}"
-zone_internal_dns: "{{guid}}.internal."
-chomped_zone_internal_dns: "{{guid}}.internal"
+subdomain_base: "{{ subdomain_base_short }}{{ subdomain_base_suffix}}"
```

Comment:
I think there should be one more space 
current: {{ subdomain_base_short }}{{ subdomain_base_suffix}}"
change requested: {{ subdomain_base_short }}{{ subdomain_base_suffix }}"

Reply:
You are right, I'll get on to it